packages/database/src/migrations/20200925051137-ReorderLaosSchoolsSchoolIndicatorsMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+import { codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const SCHOOL_INDICATORS_EIE_GROUP_CODE = 'School_Indicators_EiE';
+const SCHOOL_INDICATORS_EIE_OVERLAY_ORDERS = [
```

Comment:
Similarly for the other ones

```suggestion
const SCHOOL_INDICATORS_EIE_OVERLAYS_ORDERED = [
```